### ISSUE TITLE
1G goto line 1 fails, but any other number 2G etc works fine

### ISSUE THREAD

**Issue by rahvee** [alextsui05/VimAddin#14]
If you hit 1G, it should go to line 1, but it goes to the end of the file.

If you hit any other number, 2G, 3G, etc, it works fine.

**Comment by alextsui05** [comment]
Yes, this is an issue. It has to do with the way the `count` is recorded. Basically, the `count` is always 1 by default if you don't type in a number before the command, so in the current code, there isn't a way to disambiguate from, say, `G` and `1G`.

The workaround for this particular bug is to use `gg`, which should take you to the top of the file.
